src/main.rs

```diff
@@ -129,6 +139,15 @@ fn main() {
                 .short("i")
                 .help("Ignore non-zero exit codes."),
         )
+        .arg(
+            Arg::with_name("export-csv")
+                .long("export-csv")
+                .takes_value(true)
+                .multiple(true)
```

Comment:
Do we need to allow for multiple options here? Wouldn't all these files have the same contents?

Reply:
They would yes, however as there is no down side to writing to multiple files, I figured offering the option to write to more than one is pretty easy. I don't generally want to try and guess user's desires since we can easily write 1 or more files, in multiple formats.

Reply:
Fair point. However, I don't think that this will be an actual use case. Even if it would, the user could simply copy the file afterwards. Also, I think there is a downside to defining it `multiple = true`: the generated help text will look like `--export-csv <file>...` (or similar) which might confuse some users ("do I have to specify multiple files? one for each benchmark command?").